sorc/build_ww3prepost.sh

```diff
@@ -11,7 +11,7 @@ PDLIB="OFF"
 while getopts ":j:a:dvw" option; do
   case "${option}" in
     a) APP="${OPTARG}";;
-    d) BUILD_TYPE="DEBUG";;
+    d) BUILD_TYPE="Debug";;
```

Comment:
This also doesn't work for ww3prepost.  The flag `-DCMAKE_BUILD_TYPE=Debug` needs to be passed to `cmake` along with `-DDEBUG=ON`

Reply:
@DavidHuber-NOAA Same as above.

Reply:
See line 94: 
[[ ${BUILD_TYPE:-"Release"} = "DEBUG" ]] && MAKE_OPT+=" -DDEBUG=ON" 

I think we actually need to revert this change here and perhaps add an additional flag that @DavidHuber-NOAA  mentioned.

Reply:
@JessicaMeixner-NOAA Please open an issue and I will fix it now. This was a mistaken commit.

Reply:
I think this is working in the develop branch and not working in this PR branch and the update you made should be reverted, therefore I am not opening an issue.  Since this PR is about adding the debug option for all build scripts, we should likely check to confirm if what @DavidHuber-NOAA  mentioned is needed or not, but again I don't see a need for opening an issue.

Reply:
I tried running this with my suggested flag on Orion, but the executables failed to build.  After some debugging, I found that the cmake command
```console
cmake "${WW3_DIR}" -DSWITCH="${buildswitch}" "${MAKE_OPT}"
```
does not distinguish multiple `-D` flags within `MAKE_OPT`, so it needs to be unquoted.

Additionally, when executing `cmake` with `-DDEBUG=ON`, the following message is returned:
```console
CMake Warning:
  Manually-specified variables were not used by the project:

    DEBUG
```
So that flag should be removed.  The following commit allows for debug builds of WW3 executables: https://github.com/DavidHuber-NOAA/global-workflow/commit/14c8d6560ef1cdea5e12ee1c8c27f76e685f587b.

Reply:
Thanks for following through on this @DavidHuber-NOAA